src/lib/logger.ts

```diff
@@ -66,7 +66,13 @@ export const log = {
             // eslint-disable-next-line
             console.error(error);
 
-            if (logToAppInsights) {
+            let statusCode: number | undefined;
+
+            if ('status' in error) {
+                statusCode = error.status;
+            }
+
+            if (logToAppInsights && statusCode !== 404) {
```

Comment:
Could we use `isApiErrorWithStatus` here instead? https://github.com/BiblioNexusStudio/content-manager-web/blob/master//Users/kylegrinstead/Developer/content-manager-web/src/lib/utils/http-errors.ts#L40